spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/JdbcSearchableJobExecutionDao.java

```diff
@@ -148,7 +148,7 @@ public class JdbcSearchableJobExecutionDao extends JdbcJobExecutionDao implement
 
 	public JdbcSearchableJobExecutionDao() {
 		conversionService = new DefaultConversionService();
-		conversionService.addConverter(new StringToDateConverter());
+		conversionService.addConverter(new StringToLocalDateTimeConverter());

```

Comment:
I verified the Batch converter is functionally equivalent to the removed SCDF converter.